sdk/daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/crypto/Hash.scala

```diff
@@ -93,11 +108,19 @@ object Hash {
   private[lf] val noCid2String: Value.ContractId => Nothing =
     _ => throw HashingError.ForbiddenContractId()
 
-  private[crypto] sealed abstract class Builder {
+  private[lf] val bigIntNumericToBytes: data.Numeric => Bytes =
+    numeric => Bytes.fromByteArray(numeric.unscaledValue().toByteArray)
+
+  private[lf] val stringNumericToBytes: data.Numeric => Bytes =
+    numeric => Utf8.getBytes(data.Numeric.toString(numeric))
 
-    protected def update(a: ByteBuffer): Unit
+  private[crypto] sealed abstract class Builder(
+      numericToBytes: data.Numeric => Bytes
+  ) {
 
-    protected def update(a: Array[Byte]): Unit
+    protected def update(a: ByteBuffer, context: => String): Unit
+
+    protected def update(a: Array[Byte], context: => String): Unit
```

Comment:
add this file to the codeowners because it is now a public API where we have to be careful with changes

Reply:
Hmm where's that being tracked in this repo?

Reply:
Oh or is it a github config thing?

Reply:
https://github.com/digital-asset/daml/blob/main/CODEOWNERS - for now I would just use the daml release owners, although @remyhaemmerle-da I think we should have a separate "daml-change-owners" group to review breaking changes or changes to sensitive files.